src/site/content/en/accessible/accessible-responsive-design/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: Accessible Responsive Design
+authors:
+  - dgash
+  - megginkearney
+  - rachelandrew
+  - robdodson
+date: 2020-05-28
+description: |
+  We know that it's a good idea to design responsively to provide the best multi-device experience, but responsive design also yields a win for accessibility.
+tags:
+  - blog
+  - accessibility
+---
+
+We know that it's a good idea to design responsively to provide the best multi-device experience,
+but responsive design also yields a win for accessibility.
+
+Consider a site like <a href="https://udacity.com">Udacity.com</a>:
+
+<figure class="w-figure">
+  <img src="./udacity-100.jpg" alt="The Udacity site">
```

Comment:
```suggestion
  <img class="w-screenshot" src="./udacity-100.jpg" alt="The Udacity site">
```

I'd suggest adding the screenshot class as it'll give these images a light border. I think it helps prevent the images from looking like they're floating on the page.